src/main/java/org/skriptlang/skript/Skript.java

```diff
@@ -0,0 +1,70 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript;
+
+import org.jetbrains.annotations.ApiStatus;
+import org.skriptlang.skript.registry.SkriptRegistry;
+
+/**
+ * The main class for (soon to be) everything related to Skript. This is seperated
+ * from platform-specific implementations.
```

Comment:
```suggestion
 * The main class for everything related to Skript.
 * This is separated from platform-specific implementations.
```
I don't *think* the soon comment is necessarily needed